### PR TITLE
add translate to i18n 'es' on pagination directive.

### PR DESCRIPTION
**Pull request by bpena**
hi. 
I need this translations for pagination.

### CODE REVIEW

**Comment by PaulL1** [comment]
I think you also need to change the paginationJs to create the value `translationOf` if you want this to work.

**Comment by bpena** [comment]
If you're right. I forgot to add the change of paginationJs

**Comment by PaulL1** [comment]
I still can't see the pagination.js changes, and could you possibly squash all the commits once you're done submitting that?

File: src/features/pagination/js/pagination.js [hunk 1]
```diff
@@ -343,20 +343,21 @@
           $scope.paginationApi = uiGridCtrl.grid.api.pagination;
           $scope.sizesLabel = i18nService.getSafeText('pagination.sizes');
           $scope.totalItemsLabel = i18nService.getSafeText('pagination.totalItems');
-          
+          $scope.paginationOf = i18nService.getSafeText('pagination.of');
```
Comment (JLLeitschuh) [inline src/features/pagination/js/pagination.js]: @PaulL1 I think its right here.

**Comment by PaulL1** [comment]
Ah yes.  So it's just the squash and then we can merge?

**Comment by JLLeitschuh** [comment]
Taken care of. This also closes #3070

**Comment by JLLeitschuh** [comment]
Merged in 1338b6c901b7030ab4d20a5e7df497294e5a00f9

**Comment by bpena** [comment]
thank you for merged.
I did not have internet, so I have not answered before.

**Pull request closed without merging**